### ISSUE TITLE
Template: Button missing a text alternative - 4.1.2: Name, Role, Value

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#93]
The carrot buttons on the navigation are missing a text alternative ([FROM SITEIMPROVE](https://my2.siteimprove.com/Accessibility/59595/NextGen/Issue/1?ruleId=12&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=20&siteTarget.wcagVersion=20&siteTarget.conformanceLevels=0,1&siteTarget.issueKinds=1,2))

Buttons should always have a text label. The label can be inserted into the HTML — it does not need to be visible to pass this check.

**Comment by syarwick23** [comment]
Issue fixed with new template
